Userland/Applications/HexEditor/HexEditor.cpp

```diff
@@ -633,6 +639,12 @@ void HexEditor::paint_event(GUI::PaintEvent& event)
                 static_cast<int>(character_width()),
                 static_cast<int>(line_height() - m_line_spacing)
             };
+            Gfx::IntRect text_background_rect {
+                static_cast<int>(frame_thickness() + offset_margin_width() + bytes_per_row() * cell_width() + j * character_width() + 4 * m_padding),
+                static_cast<int>(frame_thickness() + m_line_spacing / 2 + i * line_height()),
+                static_cast<int>(character_width()),
+                static_cast<int>(line_height())
```

Comment:
Why do we static cast here but not above?

Reply:
I was just going off the code that was already there, but I will take a look.

Reply:
I cannot tell for sure why the static_casts were there in the first place, I can only hypothesize they were added to silence a warning about a narrowing conversion from size_t to int. This warning does however not occur when all of the constructor arguments are size_t, so this may have happened during https://github.com/SerenityOS/serenity/commit/4aab6ff839abbc6cbd9bee0893d453f167b47c46 when some variables were still int.

As is, the static_casts appear to be unnecessary so I have removed them.

Reply:
Thanks for investigating :)